Makefile

```diff
@@ -8,10 +8,16 @@ build-in-container: $(DEPS) clean
 		-v ${CURDIR}/build:/go/src/github.com/linuxkit/virtsock/build \
 		virtsock-build
 
-build-binaries: virtsock_stress virtsock_echo
+build-binaries: vsudd virtsock_stress virtsock_echo
 virtsock_stress: build/virtsock_stress.darwin build/virtsock_stress.linux build/virtsock_stress.exe
 virtsock_echo: build/virtsock_echo.darwin build/virtsock_echo.linux build/virtsock_echo.exe
+vsudd: build/vsudd.linux 
 
+build/vsudd.linux: $(DEPS)
+	@echo "+ $@"
+	GOOS=linux GOARCH=amd64 \
+	go build -o $@ -buildmode pie --ldflags '-s -w -extldflags "-static"' \
```

Comment:
Hm looks like we would need GO_FLAGS per platform for the other targets as well then. Could do it in another PR as it's a larger Makefile change?